server.ts

```diff
@@ -11,31 +11,69 @@ config(); //Read .env file lines as though they were env vars.
 //For the ssl property of the DB connection config, use a value of...
 // false - when connecting to a local DB
 // { rejectUnauthorized: false } - when connecting to a heroku DB
-const herokuSSLSetting = { rejectUnauthorized: false }
-const sslSetting = process.env.LOCAL ? false : herokuSSLSetting
+const herokuSSLSetting = { rejectUnauthorized: false };
+const sslSetting = process.env.LOCAL ? false : herokuSSLSetting;
 const dbConfig = {
   connectionString: process.env.DATABASE_URL,
   ssl: sslSetting,
 };
 
 const app = express();
+//const pgp = require('pg-promise')();
 
 app.use(express.json()); //add body parser to each following route handler
-app.use(cors()) //add CORS support to each following route handler
+app.use(cors()); //add CORS support to each following route handler
 
 const client = new Client(dbConfig);
 client.connect();
 
 app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
-  res.json(dbres.rows);
+  res.send("go to endpoint /pastes");
+});
+
+async function getResults() {
+  let results = [];
+  const pastebin_result = await client.query("select * from pastebins");
+  const pastebin_rows = pastebin_result.rows;
+  const summary_result = await client.query("select * from paste_summary");
+  const summary_rows = summary_result.rows;
+  results.push({
+    pastebin: pastebin_rows,
+    summary_result: summary_rows,
+  });
+  return results;
+}
+
+app.get("/pastes", async (req, res) => {
+  getResults().then((results) => res.json(results));
 });
 
+app.get("/summaries", async (req, res) => {
+  const summary_result = await client.query(
+    "select paste_summary.summary_id, pastebins.title ,paste_summary.summary from paste_summary join pastebins on paste_summary.summary_id=pastebins.id order by pastebins.sumbit_date desc limit 10"
+  );
+  res.json(summary_result.rows);
+});
+
+app.post("/pastes", async (req, res) => {
+  let createPastebin =
+    "insert into pastebins(title, text_body) values ($1, $2) ";
+  let values = req.body;
+  const createSummary =
+    "insert into paste_summary (summary_id, summary) values ((select id from pastebins order by id desc limit 1), (substring(( select text_body from pastebins order by id desc limit 1), 1, 280)) )";
```

Comment:
again this query was quite long, perhaps it could have been written on multiple lines.